main.py

```diff
@@ -12,10 +12,13 @@ def main():
     # A função abaixo abre o arquivo texto numeros.txt em modo leitura e lê as linhas dele separando
     # elas em uma lista de strings.
     with open('números.txt', 'r', encoding='utf8') as arquivo:
-        linhas_do_arquivo = arquivo.readlines()
+      lista = arquivo.readlines()
 
     # Pegue a lista de strings e converta todos os valores dentro dela para inteiros.
     # TODO COLOQUE SEU CÓDIGO AQUI E APAGUE ESSE COMENTÁRIO DEPOIS.
+    linhas_do_arquivo=[]
+    for lis in lista:
+      linhas_do_arquivo.append(int(lis))
```

Comment:
Um jeito mais simples de fazer isso é:
```suggestion
    linhas_do_arquivo = map(int, linhas_do_arquivo) # cria uma lista com todos os valores convertidos em inteiros.
```